pkg/core/native/native_neo.go

```diff
@@ -183,10 +183,22 @@ func (n *NEO) Initialize(ic *interop.Context) error {
 
 // OnPersist implements Contract interface.
 func (n *NEO) OnPersist(ic *interop.Context) error {
+	gas, err := n.GetGASPerBlock(ic, ic.Block.Index)
+	if err != nil {
+		return err
+	}
+	pubs, err := n.GetCommitteeMembers(ic.Chain, ic.DAO)
+	if err != nil {
+		return err
+	}
+	index := int(ic.Block.Index) % len(ic.Chain.GetConfig().StandbyCommittee)
+	gas.Mul(gas, big.NewInt(committeeRewardRatio))
+	n.GAS.mint(ic, pubs[index].GetScriptHash(), gas.Div(gas, big.NewInt(100)))
+
 	if !n.votesChanged.Load().(bool) {
 		return nil
 	}
-	pubs, err := n.GetValidatorsInternal(ic.Chain, ic.DAO)
+	pubs, err = n.GetValidatorsInternal(ic.Chain, ic.DAO)
```

Comment:
It'd be nice to split `GetValidatorsInternal` to not call `GetCommitteeMembers()` twice (it has some cost to it).

Reply:
Fixed (`GetCommitteeMembers` is not cached)

Reply:
But I don't see any change wrt this issue.

Reply:
Added `committee` caching.